RecoMET/METFilters/python/BadPFMuonFilter_Dz_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+BadPFMuonFilter = cms.EDFilter(
```

Comment:
there is already a BadPFMuonFilter_cfi.py with BadPFMuonFilter.
It looks like this should be called differently (```badPFMuonDz```?)
also, the cfi file is supposed to match the instance of the module defined in it. So, please rename the file (in line with the module instance name)